File: backend/src/main/kotlin/com/moa/backend/model/dto/ScoreDto.kt
package com.moa.backend.model.dto

import com.moa.backend.model.ScoreType
import com.moa.backend.model.User
import com.moa.backend.model.slim.IdeaSlimDto
import com.moa.backend.model.slim.UserSlimDto

data class ScoreDto (

    var id: Long = 0,

    var title: String,

    var score: Int,

    var type: ScoreType,

    var idea: IdeaSlimDto,

    var owner: UserSlimDto,

    )

File: frontend/src/app/models/dto/scoreDto.ts
import { IdeaSlimDto } from '../slimDto/ideaSlimDto';
import { UserSlimDto } from '../slimDto/userSlimDto';

export interface ScoreDto {
  id?: number;
  score?: number;
  type?: string;
  idea?: IdeaSlimDto;
  title?: string;
  owner?: UserSlimDto;
}
